docs/model-dev-guide/prepare-container/custom-env.rst

```diff
@@ -59,16 +59,20 @@ You can also set variables for each accelerator type, separately:
  Startup Hooks
 ***************
 
-If a ``startup-hook.sh`` file exists in the top level of your model definition directory, this file
-is automatically run with every Docker container startup. This occurs before any Python interpreters
-are launched or deep learning operations are performed. The startup hook can be used to customize
-the container environment, install additional dependencies, and download data sets among other shell
-script commands.
+If a ``startup-hook.sh`` file exists in the top level of your model definition directory, it is
```

Comment:
I think "model definition directory" may be confusing. It is only really applicable to experiments. For shells, notebooks and tensorboards, it would, I think, just be called the context directory. It may be worthwhile saying you need to supply the context directory for the NTS(not C) commands, which should be `--context`, or `-c` for each of them. I'm sure you can get it to work with `--include` as well, but requires a bit more directory management.

Reply:
done